### ISSUE TITLE
Abnormal circulating Cadherin 13 concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5990]
**Preferred term label:** Abnormal circulating Cadherin 13 concentration

**Synonyms** Abnormal T-cadherin concentration, Abnormal H-cadherin concentration

**Definition (free text, please give PubMed ID)** Concentration of Cadherin 13 in the blood is outside the normal range., PMID: 28386779

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Cadherin 13 concentration

**Synonyms** Decreased T-cadherin concentration, Decreased H-cadherin concentration

**Definition (free text, please give PubMed ID)** Concentration of cadherin 13 in the blood is below the normal range.

**Parent Term (use hpo.jax.org/app)** Abnormal circulating Cadherin 13 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Cadherin 13 concentration

**Synonyms** Increased T-cadherin concentration, Increased H-cadherin concentration

**Definition (free text, please give PubMed ID)** Concentration of cadherin 13 in the blood is above the normal range.

**Parent Term (use hpo.jax.org/app)** Abnormal circulating Cadherin 13 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I do not think that  Cadherin 13 is found in the blood circulation and as far as I know there is now clinical test for it.  PMID: 28386779 does not talk about blood levels. Can you please specify?

**Comment by pnrobinson** [comment]
No normal ranges have been determined for this protein in serum. For instance, there is no LOINC term for this test. Therefore, it is possible to do a cohort study that shows a higher level in individuals with some condition, but it is not possible to do a test in an individual and say that the level is "too high" or "too low". This means that this would not be a valid HPO term.
